--------------------------------------
Running PBS prologue script
--------------------------------------
User and Job Data:
--------------------------------------
Job ID:    962396.smic3
Username:  coltonb
Group:     Users
Date:      27-Jun-2023 03:58
Node:      smic118 (19168)
--------------------------------------
PBS has allocated the following nodes:

smic118

A total of 20 processors on 1 nodes allocated
---------------------------------------------
Check nodes and clean them of stray processes
---------------------------------------------
Checking node smic118 03:58:17 
Done clearing all the allocated nodes
------------------------------------------------------
Concluding PBS prologue script - 27-Jun-2023 03:58:17
------------------------------------------------------
Loading mvapich2/2.3.3/intel-19.0.5
  Loading requirement: intel/19.0.5
JOBS_PER_NODE=1
/home/coltonb/JCMsuite_5.4.0/ThirdPartySupport/Python/bin/python3.9: can't open file '/ddnA/work/coltonb/Multivariable_Data_Struct/Near_Field_60AOI_280wvl_45azi/run_sum.py': [Errno 2] No such file or directory
------------------------------------------------------
Running PBS epilogue script    - 27-Jun-2023 03:58:17
------------------------------------------------------
Checking node smic118 (MS)
Checking node smic118 ; cleared cache ; ok
------------------------------------------------------
Concluding PBS epilogue script - 27-Jun-2023 03:58:20
------------------------------------------------------
Exit Status:     2
Job ID:          962396.smic3
Username:        coltonb
Group:           Users
Job Name:        near_field_60aoi
Session Id:      19167
Resource Limits: ncpus=1,neednodes=1:ppn=20,nodes=1:ppn=20,walltime=01:00:00
Resources Used:  cput=00:00:00,mem=0kb,vmem=0kb,walltime=00:00:03
Queue Used:      workq
Account String:  hpc_mcpeak11
Node:            smic118
Process id:      19867
------------------------------------------------------
